lib/internal/Magento/Framework/App/Action/Action.php

```diff
@@ -24,6 +24,7 @@
  * Action classes that do not extend from this class will lose this behavior and might not function correctly
  *
  * @deprecated Use \Magento\Framework\App\ActionInterface
+ * @see https://community.magento.com/t5/Magento-DevBlog/Decomposition-of-Magento-Controllers/ba-p/430883
```

Comment:
Please change to 
```
/**
 * @deprecated Inheritance was eliminated, composition should be used instead.
 * @see \Magento\Framework\App\ActionInterface
 */
```

Reply:
I would keep both links, just to have more info

Reply:
It's forbidden to add links to code

Reply:
Could you explain where it declared and why?
From my perspective add link that could describe more detailed why class is deprecated and what to use instead is really good option!

Reply:
@ihor-sviziev we can put links to official documentation only.

Reply:
Got your point. Thx